lib/meals.js

```diff
@@ -1,5 +1,82 @@
 'use strict';
 
 module.exports = {
-  // your code goes here
+
+    name: 'Celena',
+    age: '26',
+    calorieTarget: 2000,
+    meals: [
+            {
+              title: 'Breakfast',
+              date: '2016-21-01',
+              desc: 'Eggs and bacon',
+              estCalories: 500
+            },
+            {
+              title: 'Lunch',
+              date: '2016-21-01',
+              desc: 'Curry and rice',
+              estCalories: 800
+            },
+            {
+              title: 'Snack',
+              date: '2016-21-01',
+              desc: 'Granola bar',
+              estCalories: 100
+            },
+            {
+              title: 'Dinner',
+              date: '2016-21-01',
+              desc: 'Dumplings',
+              estCalories: 200
+            },
+            {
+              title: 'Breakfast',
+              date: '2016-22-01',
+              desc: 'Cereal and fruit',
+              estCalories: 400
+            },
+            {
+              title: 'Lunch',
+              date: '2016-22-01',
+              desc: 'Pork curry udon',
+              estCalories: 1000
+            },
+            {
+              title: 'Dinner',
+              date: '2016-22-01',
+              desc: 'Mushroom sandwich',
+              estCalories: 600
+            }
+          ],
+    caloriesEatenOn: function(date) { // calculates total calories eaten on specific day
+      let totalCalories = 0;
+      for (let i = 0; i < this.meals.length; i++) {
+        if (this.meals[i].date === date) {
+          totalCalories += this.meals[i].estCalories;
+        }
+      return totalCalories;
+    }
+  },
+
+    avgDailyCalories: function(date) { // calculates avg calorie intake on specific day
+      let mealCount = 0;
+      for (let i = 0; i < this.meals.length; i++) {
+        if (this.meals[i].date === date) {
+          mealCount++;
+        }
+      }
```

Comment:
Nice solution! Next, try extracting `mealCount` into it's own method, and then reuse it here to make this method body really easy to read.